index.js

```diff
@@ -191,6 +191,28 @@ export default class Supercluster {
         return expansionZoom;
     }
 
+    getPointUnclusterZoom(point) {
+        const [lng, lat] = point;
+        const pointX = fround(lngX(lng));
+        const pointY = fround(latY(lat));
+
+        let expansionZoom = this.options.minZoom;
+        while (expansionZoom < this.options.maxZoom) {
+            const tree = this.trees[expansionZoom];
+
+            const pointIdxs = tree.within(pointX, pointY, 0);
+
+            const unclustered = pointIdxs.some(
+                idx => tree.data[idx].parentId !== -1
```

Comment:
This looks wrong — `tree.data` layout is different now as I mentioned, it's a flat array of numbers. It should look something like this:

```suggestion
                idx => tree.data[this.stride * idx + OFFSET_PARENT] !== -1
```

But even that, this doesn't look right, because `parentId` will not be `-1` for ALL points that eventually end up in a cluster regardless of zoom level, so this logic doesn't work here. Instead I'd suggest checking whether number of points equals 1 (`tree.data[this.stride * idx + OFFSET_NUM] === 1`).